packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -676,11 +676,9 @@ class _DecoyChildState extends State<_DecoyChild> with TickerProviderStateMixin
 
     _boxDecoration = DecorationTween(
       begin: const BoxDecoration(
-        color: Color(0xFFFFFFFF),
```

Comment:
If we need to access the `_boxDecoration` property used in the `_DecoyChild` directly in the tests, I would need to expose it somehow or at least write a public method to validate its state in tests, as it's currently a private property.

It means I would need to
 
- make `_DecoyChild` public to Test its state while the animation is in progress.
- write `validateDecoyColor` public predicate method to use it in tests to verify its state.

I have to introduce a new public class by publicizing `_DecoyChild` or put them in a new file and not export them in Cupertino library.

Reply:
I think you could try finding the `Container` that is built by the ` _buildAnimation()`, using a Finder, in the test?

Reply:
To avoid brittle tests, I prefer making `_DecoyChild` available by putting it in a new file that's not included in the Cupertino library so that it's available in tests but not visible to the library users as before, and now I can find it by Type and find its state & verify its color.

Could you please review my last commit 42b62372fde027f3f0c1c1e319de895abeb0bc38, where I implemented this?

If that looks good, I can follow up with a clean-up commit to fix dart analysis so that it's ready to merge.